ocaml/sdk-gen/csharp/autogen/src/JsonRpc.cs

```diff
@@ -207,12 +207,12 @@ private T Rpc<T>(string callName, JToken parameters, JsonSerializer serializer)
             webRequest.KeepAlive = KeepAlive;
             webRequest.UserAgent = UserAgent;
             webRequest.ConnectionGroupName = ConnectionGroupName;
-            webRequest.ProtocolVersion = ProtocolVersion;
+            webRequest.ProtocolVersion = ProtocolVersion ?? webRequest.ProtocolVersion;
             webRequest.ServicePoint.Expect100Continue = Expect100Continue;
             webRequest.AllowAutoRedirect = AllowAutoRedirect;
```

Comment:
While debugging I noticed that in Xmlrpc the default for AllowAutoRedirect was true. I'm not certain whether it is needed for the server API, but let's keep it as it was to be on the safe side. The place to set it in when we create the Jsonrpc client in the Session constructor.